src/Matcher_lexer.mll

```diff
@@ -64,6 +64,22 @@ rule token = parse
 | "del" { BINDING (Delete) }
 | "delete" { BINDING (Delete) }
 | "bs" { BINDING (Backspace) }
+| "lt" { BINDING (LessThan) }
+| "gt" { BINDING (GreaterThan) }
+| "minus" { BINDING (Minus) }
+| "plus" { BINDING (Plus) }
+| "(" { BINDING (LeftParen) }
+| ")" { BINDING (RightParen) }
+| "|" { BINDING (Pipe) }
+| "_" { BINDING (Underscore) }
+| "%" { BINDING (Percent) }
+| "+" { BINDING (Plus) }
+| "-" { BINDING (Minus) }
+| "=" { BINDING (Equals) }
+| "//<" { BINDING (LessThan) }
+| "//>" { BINDING (GreaterThan) }
```

Comment:
Is this intended to be escaping? If so, the slash usually goes the other way.

Reply:
Ouch, thanks for catching this!